### PR TITLE
libsystemd: Add missing memory pressure functions to public symbols

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

**Comment by poettering** [comment]
lgtm

**Review by yuwata — APPROVED** [review]

File: src/libsystemd/libsystemd.sym [hunk 1]
```diff
@@ -819,4 +819,8 @@ global:
         sd_session_get_start_time;
         sd_uid_get_login_time;
         sd_pid_notifyf_with_fds;
+        sd_event_add_memory_pressure;
+        sd_event_source_set_memory_pressure_type;
+        sd_event_source_set_memory_pressure_period;
+        sd_event_trim_memory;
```
Comment (yuwata) [inline src/libsystemd/libsystemd.sym:825]: By the way, why these missing symbols are not detected by CIs ?? @mrc0mmand

Comment (daandemeyer) [reply src/libsystemd/libsystemd.sym:825]: I only discovered them by chance in #27442, definitely seems to be something missing from our coverage. Maybe we could write some unit test for this?

Comment (yuwata) [reply src/libsystemd/libsystemd.sym:825]: I thought we had a test about such, but not sure, maybe my memory is broken.

Comment (yuwata) [reply src/libsystemd/libsystemd.sym:825]: Ah, I mean, test-libsystemd-sym.

**Pull request merged**